source/packages/libraries/clients/assetlibrary-client/src/client/templates.lambda.service.ts

```diff
@@ -68,8 +68,13 @@ export class TemplatesLambdaService extends TemplatesServiceBase implements Temp
         ow(resource.templateId,'templateId', ow.string.nonEmpty);
         ow(resource.category,'category', ow.string.nonEmpty);
 
+        const templateId = resource.templateId;
```

Comment:
What is the reasoning behind this change ?!

Reply:
You're right - no real functionality change here, but it is consistent with the `createTemplate` functionality. When developing, it was a little confusing when creating/updating templates to see one delete your `resource.templateId`/`resource.category`, but the other one did not.